src/components/multiselect-list.cjsx

```diff
@@ -140,7 +140,7 @@ class MultiselectList extends React.Component
 
   _onClickItem: (item, event) =>
     return unless @state.handler
-    if event.metaKey
+    if event.metaKey || event.ctrlKey
```

Comment:
The `Event` object has a `ctrlKey` property, I think this should be standard on all platforms but in case I put it after the or operator as to not affect existing behaviour.